modules/ydn.py

```diff
@@ -0,0 +1,13 @@
+from .base import Module
+import requests
+from bs4 import BeautifulSoup
+import random
+
+
+class YDN(Module):
```

Comment:
Oh querying the RSS, that's clever. I was planning to do a search thing, but this works pretty well for now.

Reply:
I updated it to do the search